lib/global-admin/addon/catalog/template.hbs

```diff
@@ -14,7 +14,7 @@
     <div>
       {{#if library}}
         <div class="btn-group no-inline-space">
-          <button class="btn btn-link btn-sm bg-default" {{action "disableLibrary"}} disabled={{togglingLibrary}}>{{#if togglingLibrary}}<i class="icon icon-spinner icon-spin"></i> {{/if}}{{t 'generic.disable'}}</button>
+          <button class="btn btn-link btn-sm bg-default" {{action "disableLibrary"}} disabled={{disabledLibrary}}>{{#if togglingLibrary}}<i class="icon icon-spinner icon-spin"></i> {{/if}}{{t 'generic.disable'}}</button>
```

Comment:
Add `waitForTransition()` after the save calls in the disable actions (`[...].save().waitForTransition().[...]`), instead of adding new disabled computed properties.  Then `toggling[thing]` will just stay true until the remove is complete, and you'll disable the button and show the spinner instead of showing a disabled button that has no explanation and no indication of activity.